Packs/FeedNVDv2/Integrations/FeedNVDv2/FeedNVDv2.py

```diff
@@ -0,0 +1,421 @@
+        last_run = parse(last_run_data.get("lastRun"))
+
+    if command == 'nvd-get-indicators':
+        history = demisto.getArg('history')
+        demisto.debug(f'Retrieving last {history} days of CVEs using nvd-get-indicators')
+        last_run = (datetime.now() - timedelta(days=int(history)))
+    # First run of the integration
+    elif "lastRun" not in last_run_data or test_run:
+        last_run = start_date
+    # last_run is present so parse last date
+    else:
+        last_run = parse(last_run_data.get("lastRun"))
+
+    last_mod_start_date = last_run
+    last_mod_end_date = datetime.now()  # type: ignore[attr-defined]
+
+    while exceeds_span and last_mod_end_date and last_mod_start_date:
+        delta = (last_mod_end_date - last_mod_start_date).days  # type: ignore[TypeError]
+        if delta > 120:
+            last_mod_end_date = last_mod_start_date + timedelta(days=120)  # type: ignore
+        else:
+            exceeds_span = False
+
+        iteration += 1
+
+        raw_cves = retrieve_cves(client, last_mod_start_date, last_mod_end_date, False)
+
+        demisto.debug(print(f'\n\nlastModStartDate: {last_mod_start_date.strftime(DATE_FORMAT)}'  # noqa: T201
+                            + f'\nlastModEndDate: {last_mod_end_date.strftime(DATE_FORMAT)}'
+                            + f'\nFetch Iteration: {str(iteration)}' + '\nCurrent Total Fetched Indicator Count: '
+                            + f'{str(len(parsed_cves))}\n\n'))
+
+        if raw_cves:
+            temp_cves = build_indicators(client, raw_cves)
+            demisto.createIndicators(temp_cves)
+
+            parsed_cves += temp_cves
+            temp_cves.clear()
+
+        raw_cves.clear()
+
+        last_mod_start_date = last_mod_end_date
+        last_mod_end_date = datetime.now()
+
+    demisto.setLastRun({"lastRun": datetime.now
+                        (timezone.utc).isoformat()})
+
+
+def main() -> None:
+    """
+    Main integration function that defines the client object and initiates calls to
+    the user-specified integration command
+
+    Args:
+        None
+
+    Returns:
+        None
+
+    """
+
+    params = demisto.params()
+    base_url: str = "https://services.nvd.nist.gov"  # disable-secrets-detection
+    verify_cert = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    api_key = params.get('apiKey').get('password')
+    tlp_color = params.get('tlp_color')
+    has_kev = params.get('hasKev')
+    start_date = params.get('start_date')
+    feed_tags = params.get('feedTags')
+    command = demisto.command()
+    demisto.debug(f'Command being called is {command}')
+    try:
+        client = Client(
+            base_url=base_url,
+            verify=verify_cert,
+            proxy=proxy,
+            api_key=api_key,
+            tlp_color=tlp_color,
+            has_kev=has_kev,
+            start_date=start_date,
+            feed_tags=feed_tags
+        )
+
+        if command != "test-module":
```

Comment:
@AradCarmi - Isn't the opposite better here?

If command == "test-module":
    test module...
else:
    fetch...